tbx/static_src/sass/components/_avatar.scss

```diff
@@ -2,12 +2,12 @@
 
 .avatar {
     position: relative;
-    width: 98px;
-    height: 98px;
+    width: 72px;
```

Comment:
As mentioned I've reduced the default mobile size of the avatar here, but we will probably need some more variants